syntax = "proto3";

package Commands;

option java_package = "com.interfaces.proto";
option java_outer_classname = "UserCommandToRunProto";

message UserCommandToRun
{
    CoordinateSystem coordinate_system = 1;
}

message CoordinateSystem
{
    int32 x_coordinate = 1;
    int32 y_coordinate = 2;
}
